Animations/Animation.js

```diff
@@ -0,0 +1,126 @@
+// 'use strict';
+var React = require("react/addons");
+var Arrival = require('Arrival');
+var Easings = require("./AnimationEasings.js");
+var ReactTransitionGroup = React.addons.TransitionGroup;
+
+//React.createClass FUNCTION: function to create animation component class
```

Comment:
Move the comments about the React.createClass call within the makeTransition function.